chartpress.py

```diff
@@ -15,11 +15,26 @@
 
 __version__ = '0.2.0.dev'
 
+# name of the environment variable with GitHub token
+GITHUB_TOKEN_KEY = 'GITHUB_TOKEN'
```

Comment:
You've defined this variable, but haven't used it below. Since it's used only once, I think we can drop the variable.

Reply:
Thanks!